WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionButton.swift

```diff
@@ -155,7 +155,7 @@ extension ReaderSortingOptionButton: Accessible {
 
         // TODO: check if accessibility is set correctly
         accessibilityIdentifier = "Reader sorting option button"
-        accessibilityLabel = NSLocalizedString("Sorting option button", comment: "Accessibility label for sorting option button")
```

Comment:
Removed "button" from the label because it appears that VoiceOver ignores the word "button" when it is speaking, probably because it speaks "button" at the and as a result of `accessibilityTraits = UIAccessibilityTraits.button`